php/templates/login.twig

```diff
@@ -8,7 +8,7 @@
             {% if is_login_allowed == true %}
                 <p>Log in using your Nextcloud AIO password:</p>
                 <form method="POST" action="/api/auth/login" class="xhr">
-                    <input type="text" autocomplete="off" name="password" placeholder="Password" />
+                    <input type="password" autocomplete="current-password" name="password" placeholder="Password" wfd-id="id0" id="Password" oninput="showPassword()">

```

Comment:
```suggestion
                    <input type="passwordInput" autocomplete="current-password" name="password" placeholder="Password" id="Password" oninput="showPassword('passwordInput')">

```

Reply:
why do you change the type? you need to change the id

Reply:
```suggestion
                    <input type="password" autocomplete="current-password" name="password" placeholder="Password" id="Password" oninput="showPassword('Password')">

```

Reply:
```suggestion
                    <input type="password" autocomplete="current-password" name="password" placeholder="Password" 
                    id="passwordInput" oninput="showPassword('passwordInput')">

```

Reply:
```suggestion
                    <input type="password" autocomplete="current-password" name="password" placeholder="Password" id="passwordInput" oninput="showPassword('passwordInput')">

```